torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -48,6 +50,10 @@ int& getTECudaPointwiseBlockSize() {
   return te_cuda_pointwise_block_size;
 }
 
+bool& getTEGenerateBlockCode() {
```

Comment:
Look at my other commend on this flag. I would strongly prefer to use the input device, instead of a flag to get into this mode. A TODO might be fine for now to get this unblocked.

Reply:
I added a TODO for now. I am going to revisit this later. Thanks for the design inputs!